composer.json

```diff
@@ -73,6 +72,7 @@
     "neitanod/forceutf8": "^2.0.4",
     "nesbot/carbon": "^2.27",
     "pear/net_url2": "^2.2",
+    "google/apiclient": "^2.12",
```

Comment:
```suggestion
```

Reply:
it's removed on line 64 but re-added on 75, should be enough here https://github.com/pimcore/pimcore/blob/5fa71a342b8f070617a3f848d2a5365e3e3c2f7c/bundles/GoogleMarketingBundle/composer.json#L18

Reply:
Yes we have to remove it before merging, but testing without it being in the core is not possible as long as the bundle is still in the bundles. Once we move it into it's own repo we  can safely remove this line.